.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -16,6 +16,10 @@ public function setUp() {
 
 		set_current_screen( 'dashboard' );
 
+		// Reset queued scripts and styles.
+		global $wp_scripts, $wp_styles;
+		$wp_scripts = new WP_Scripts();
+		$wp_styles = new WP_Styles();
```

Comment:
Should we add a blank line after `global $wp_scripts, $wp_styles;`?

The `=` signs should also be aligned here.